Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM golang:1.20
 
 WORKDIR /app
 
-COPY go.mod ./
+COPY go.mod go.sum ./
```

Comment:
Nice catch, you may not have noticed it, but this method actually helps with docker layer caching.